.expeditor/scripts/shared_release_habitat.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
This is just a random thought... would it be worth prefixing the names of functions in this (and similar) shared "library files" with the "namespace" the function comes from, as an additional clue to readers where code is defined and how it is intended to be used?

e.g., `release.import_keys`, `release.get_latest_pkg_version`, etc?

I'd know exactly where the function was defined, and if I'm using a `release.*` function in one of my end-to-end tests, I know I've done something very wrong.